examples/oic.r.acl-example2.json

```diff
@@ -29,61 +29,6 @@
           }
         ]
       }
-    ],
```

Comment:
Re: "aces" still exists in ACL(1). From oic.r.acl.json:

`  "definitions": {
    "oic.r.acl": {
      "type": "object",
      "properties": {
        "aclist": {
          "type": "object",
          "description": "Subject-based Access Control Entries in the ACL resource",
          "properties": {
            "aces": {
              "type": "array",
              "items": {
                "$ref": "oic.sec.ace.json#/definitions/oic.sec.ace"
              }
            }
          },
          "required": [ "aces" ]
        },
        "rowneruuid": {
          "description": "The value identifies the unique resource owner",
          "$ref": "../../core/schemas/oic.types-schema.json#/definitions/uuid"
        }
      }
`

The only effective change that should have been made to ACL(1) was to remove "aces2" from the "aclist" object. (a lot of nasty schema stuff stuff was also removed in the process - but nothing that should affect the allowed forms)